### ISSUE TITLE
[PROPOSAL] Change Shortest Path Selection to Shift Ctrl Click

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#989]
From the BM inbox, added here so I don't forget:

> Yes, as you know, Ctrl is reserved for creating/slicing geometry but how about " Ctrl + Shift"?
> 
> In my case, Ctrl + Shift + left click works well and feel no problem. How about trying this setting at the attached image (bellow)?
> 
> Only with the knife tool, it didn't seem to work properly with my settings, but with other tools the preview appears with holding the Ctrl key, but with holding Ctrl + Shift + mouse dragging the preview disappears so what's the problem? There is no problem.
> 
> For Blender users, if the operation of selecting shortest path is same between Blender and Retopoflow,  it's useful. And  I recommend Ctrl + Shift + single-click settings for my friends, blog readers, and social media followers. In fact I haven't felt any problems so far. 
> 
> I'm happy if you consider changing the hotkey about it but this is only my review opinion, not fatal bug. 
> 
> ![image](https://user-images.githubusercontent.com/12982300/115929228-4c4c6480-a43c-11eb-8442-2888cd79a5f5.png)

I'll have to spend more time testing this out, but it sounds like a reasonable proposal since it more closely matches Blender's shortest path selection.

**Comment by vxlcoder** [comment]
i'm fine with this change.  I made some tweaks to the way the insert previs works, so this keymapping now works fine with knife and other tools.

on a related note: the insert alt0 and insert alt1 keymaps seem to not be used anymore.  they were used for alternate insertions for certain tools (ex: tri- or edge-only insert in PP), but now we have different insert modes to handle these (which I like a lot more).

should we move forward with the change?  we'll need to inform previous customers that this is now changed
